Changed intake constants for intake speed & intake angles

ananyakidambi opened the pull request.

ananyakidambi committed:
Merge remote-tracking branch 'origin/main' into akidambi

ananyakidambi committed:
Merge remote-tracking branch 'origin/main' into akidambi

ananyakidambi committed:
changed constants to make intake faster

ananyakidambi committed:
Changed constants to make intake faster

zaraansari17 committed:
updated min and max angles

ananyakidambi committed:
Changed expel note time

jmullins16 commented on Comp2024/src/main/java/frc/robot/subsystems/Intake.java:
```diff
@@ -89,8 +89,8 @@ private enum RotaryMode
 
-  private static final double        kRotaryAngleMin       = -179.27;
-  private static final double        kRotaryAngleMax       = 21.9;
+  private static final double        kRotaryAngleMin       = kRotaryAngleRetracted - 2;
```
Can we make these "offsets" that define the min and max both be some even number of degrees--like 3.0--instead of something like 2.39. Negative and positive offsets should be about the same "few" (3.0) degrees since they serve the same purpose.

jmullins16 reviewed (dismissed):
Approving, but please make the min/max numbers a little more consistent.

jmullins16 committed:
Added in full structure for intake and feeder tuning
- needs feeder measurements for all three positions
- needs intake handoff position tested

ananyakidambi committed:
Merge remote-tracking branch 'origin/main' into akidambi

zaraansari17 reviewed (dismissed):
Looks good

grace-mills reviewed (dismissed):
all good

ananyakidambi committed:
Fixed comment table for angles

ananyakidambi committed:
Offset and practice robot angles update
